From 852eec102d9793d7c378ccfb582e641cff942ed1 Mon Sep 17 00:00:00 2001
From: Zongdong Jiao <zongdong.jiao@amlogic.com>
Date: Wed, 5 Aug 2015 14:57:20 +0800
Subject: [PATCH 5746/5965] PD#110571: hdmitx20: remove kthread_stop

Paint a snake with feet as call kthread_stop() at hdmitx
reboot_notifier.

Change-Id: I152977725e502fc3c6cadf77934fe1c6d783aace
---
 drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c b/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c
index cffedbbac6ff..165d7aceee4c 100644
--- a/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c
+++ b/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c
@@ -123,8 +123,6 @@ static int hdmitx_reboot_notifier(struct notifier_block *nb,
     hdmitx_dev_t *hdev = container_of(nb, struct hdmi_tx_dev_s, nb);
     hdev->HWOp.Cntl(hdev, HDMITX_AVMUTE_CNTL, AVMUTE_SET);
     mdelay(25);
-    if (hdev->task)
-        kthread_stop(hdev->task);
     return NOTIFY_OK;
 }
 
-- 
2.19.0

